app/views/pages/rescript.html.erb

```diff
@@ -1 +1 @@
-<%= react_component "RescriptShow", prerender: true %>
+<%= react_component "RescriptShow", prerender: false %>
```

Comment:
Why turn off SSR?

Reply:
Sorry it is an initial implementation, it is not done yet. I'll ping you when it is done.